sdk/cosmos/tests.yml

```diff
@@ -1,7 +1,7 @@
 trigger: none
 
 jobs:
```

Comment:
You need this top level `jobs` field to be renamed to `stages` too for all these `tests.yml` files. Looks like you did it for a few services but not for the majority.